### DISCUSSION TITLE
新型コロナが収束した場合について

### DISCUSSION THREAD

**Discussion by Takym** [cocoa-mhlw/cocoa#247] [Q&A] [answered]
新型コロナが収束した場合は本アプリはどうなるのでしょうか？

リポジトリをアーカイブして開発を中断するのか、又は新たに感染爆発が発生した際に再び利用できる状態にしておくのでしょうか？

**Comment by keiji** [comment, marked as answer]
初Questionですね！ 🎉 

そして、本リポジトリの目的というか範囲は技術的なものに限定しています。Discussionsも同様なので、質問についてもその範囲でお願いします……！
（技術的なものか判別がつきにくい質問などについては、情報共有できることがあればしますが、必ず答えが返ってくるわけではないと言うことで、何卒よろしくお願いします）
